editor/src/clj/editor/code/data.clj

```diff
@@ -2426,7 +2426,8 @@
 
     :else
     (when-some [clickable-region (some (fn [region]
-                                         (when (and (some? (:on-click! region))
+                                         (when (and (or (some? (:on-click! region))
+                                                        (= :diagnostic (:type region)))
```

Comment:
It feels a bit strange having a special case for `:diagnostic` here. Perhaps we should treat hover popups as a higher-level concept?

Reply:
I agree that we should disentangle diagnostics from hovers.